server/src/test/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryApplicationTest.java

```diff
@@ -1,4 +1,90 @@
 package org.opentosca.toscana.plugins.cloudfoundry;
 
-public class CloudFoundryApplicationTest {
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+import org.opentosca.toscana.core.BaseJUnitTest;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class CloudFoundryApplicationTest extends BaseJUnitTest{
+    
+    private final static CloudFoundryApplication myApp = new CloudFoundryApplication();
+    
+    @Before
+    public void setUp() {
+        String appName = "testApp";
+        myApp.setAppName(appName);
+        
+        myApp.addEnvironmentVariables("TEST1");
+        myApp.addEnvironmentVariables("TEST2","TEST_VALUE2");
+        myApp.addEnvironmentVariables("TEST3");
+        
+        myApp.addService("TEST_SERVICE1");
+        myApp.addService("TEST_SERVICE2");
+        
+        myApp.addFilePath("/path/to/test");
+        
+        myApp.addAttribute("TEST_ATTRIBUTE1", "ATTRIBUT1");
+        myApp.addAttribute("TEST_ATTRIBUTE2", "ATTRIBUT2");
+        
+        myApp.addBuildpack("TEST_BUILDPACK");
+    }
+    
+    @Test
+    public void getAppName() {
+        assertEquals("testApp", myApp.getAppName());
+    }
+    
+    @Test
+    public void getEnvironmentVariables() {
+        Map<String, String> env = myApp.getEnvironmentVariables();
+        
+        assertTrue(env.size() == 3);
+        assertEquals("TEST_VALUE2",env.get("TEST2"));
+    }
+
+    @Test
+    public void getServices () {
+        List<String> service = myApp.getServices();
+
+        assertEquals("TEST_SERVICE1",service.get(0));
```

Comment:
Many strings are coded twice.  this is quite error-prone (e.g., when rewording string content).
If you need strings both in test setup and for assertion, it's best practice to introduce a field.